cfg/tools/train_siamese.yaml

```diff
@@ -3,30 +3,38 @@
 #####################################
```

Comment:
A few points about configs:
1. We usually try to keep a single example config, such as `train_siamese.yaml` in this case, that we don't modify. If you have a new config for nearest neighbors that is working, create a new one such as `train_siamese_nn.yaml`.
2. If this config isn't 100% working (like if you're just testing stuff), don't commit the changes.

This just helps us keep the configs in the master branch clean so that we can easily reference and replicate things.